packages/shared/src/hooks/useFeedLayout.ts

```diff
@@ -92,7 +93,8 @@ const getFeedPageLayoutComponent = ({
 export const useFeedLayout = ({
   feedRelated = true,
 }: UseFeedLayoutProps = {}): UseFeedLayoutReturn => {
-  const isLaptop = useViewSize(ViewSize.Laptop);
+  const isLaptopSize = useViewSize(ViewSize.Laptop);
+  const isLaptop = isNullOrUndefined(isLaptopSize) ? true : isLaptopSize;
```

Comment:
```suggestion
  const isLaptop = isNullOrUndefined(isLaptopSize) || isLaptopSize;
```

Reply:
The issue about this evaluation is that it will always be true at first since on load, this is null or undefined.